{{cookiecutter.project_slug}}/_/ci-services/definitions/.gitlab-ci.yml

```diff
@@ -81,6 +81,9 @@ stages:
   - oc tag "${SOURCE}/${CI_PROJECT_NAME}:${CI_COMMIT_SHA}"
            "${TARGET}/${CI_PROJECT_NAME}:${CI_COMMIT_SHA}"
 {%- endif %}
+  - for OBSOLETE in $(oc get configmap --sort-by='.metadata.creationTimestamp' -o name
+      -l component=application,environment=${CI_ENVIRONMENT_NAME} | tail -n +5);
```

Comment:
I think we should add the `app={{ cookiecutter.project_slug }}` label to really make sure not to delete any other ConfigMaps.

Reply:
We have a kind of mixed use of `{{ cookiecutter.project_slug }}` and `${CI_PROJECT_NAME}`. I tried to consolidate this somewhat, for both Bitbucket Pipelines and GitLab CI.